docs/rules/no-restricted-paths.md

```diff
@@ -37,3 +37,43 @@ The following patterns are not considered problems when configuration set to `{
 ```js
 import baz from '../client/baz';
 ```
+
+---------------
+
+Given the following folder structure:
+
+```
+my-project
+├── client
+│   └── foo.js
+│   └── baz.js
+└── server
+    ├── one
+    │   └── a.js
+    │   └── b.js
+    └── two
+```
+
+and the current file being linted is `my-project/server/one/a.js`.
+
+and the current configuration is set to:
+
+```
+{ "zones": [ {
+    "target": "./tests/files/restricted-paths/server/one",
+    "from": "./tests/files/restricted-paths/server",
+    "except": ["./tests/files/restricted-paths/server/one"]
```

Comment:
It seems like these should be relative paths from the “from”, and be disallowed from backtracking out of it - it wouldn’t make sense to have an “except” that wasn’t included in the “from” in the first place.